tests/test_vis.py

```diff
@@ -582,3 +582,23 @@ def test_intent_override_all_column():
     assert (
         "y = alt.Y('Record', type= 'quantitative', title='Number of Records'" in current_vis_code
     ), "All column not overriden by intent"
+
+
+def test_abbrev_title():
+    long_content = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    dataset = [
+        {"long_attr": long_content, "normal": 3, "normal2": 1},
+        {"long_attr": long_content, "normal": 3, "normal2": 1},
+        {"long_attr": long_content, "normal": 2, "normal2": 1},
+        {"long_attr": long_content, "normal": 4, "normal2": 1},
+    ]
+    df = pd.DataFrame(dataset)
+    lux.config.plotting_backend = "matplotlib"
+    vis = Vis(["normal2", "normal", f"long_attr={long_content}"], df)
+    vis_code = vis.to_matplotlib()
+    print(vis_code)
+    assert "long_attr = Lor...t laborum.'" in vis_code
+
+    vis_code = vis.to_altair()
+    print(vis_code)
+    assert "long_attr = Lor...t laborum.'" in vis_code
```

Comment:
```suggestion
    assert "long_attr = Lor...t laborum.'" in vis_code
    lux.config.plotting_backend = "altair"
```

Reply:
The sql test errors seem to be resulting from the fact that matplotlib plotting backend is set here and used throughout. Note that the "lux" variable persists throughout the entire test session, so any configs should set this back to the default value, so that it doesn't affect other tests suites outside of this function.